README.md

```diff
@@ -140,11 +142,10 @@ For Example: The component [**`<DAO address="0x...">`**](./src/components/DAO.ts
 
 Note: ComponentLists break this rule, but will be covered further down.  
 
-## Component Contexts (Data, Code, Prose)  
-Each component has 3 core contexts:  
+## Component Contexts (Data)  
+Each component has one core context:  
 * Data - GraphQL Schema  
-* Code - Abstracted Protocol Functionality  
-* Prose - TBD  
+* NOT Code - Abstracted Protocol Functionality (?)
```

Comment:
The abstracted protocol functionality will now come from data I think? Just want to make sure to update the readme sir! @dOrgJelli 

Like I said before, please correct me if i'm wrong